tests/vmi_networking_test.go

```diff
@@ -748,6 +748,9 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					Expect(tests.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
 				}
 
+				By(fmt.Sprintf("Checking ping from clientVMI console to serverVMI(%s)", serverIP))
+				Expect(tests.PingFromVMConsole(clientVMI, serverIP)).To(Succeed())
```

Comment:
do we want to do the opposite ping as well ? (unless we done it already but i am not sure we did)

Reply:
Good catch I will add it.

Reply:
Done.

Reply:
Done.

Reply:
where is it done? I cannot fins it.

Reply:
https://github.com/kubevirt/kubevirt/pull/4093#discussion_r482732892
see here please @AlonaKaplan 
it was removed (i am against removing it)